core/src/errors.rs

```diff
@@ -1,94 +1,41 @@
-use crate::utils::string::*;
-use core::fmt;
+use alloc::string::String;
+use miden_formatting::hex::DisplayHex;
 
 // INPUT ERROR
 // ================================================================================================
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, thiserror::Error)]
 pub enum InputError {
+    #[error("{0} is not a valid field element: {1}")]
+    NotFieldElement(u64, String),
```

Comment:
I don't recall explicitly moving this - my suspicion is that this happened when I rebased on changes merged into `next`, as I believe there was at least one commit that contained a lot of formatting tweaks and such. The diff doesn't reflect that of course, but the original commit I based this on must have had them in this order, because I wouldn't have had any reason to move this variant explicitly.

I suspect that what happened was that I was hand-writing the resolution for the merge conflict and happened to base the resolution on the old enum layout, rather than the new one.

In any case, I can make these alphabetically sorted again.